docs/source/image_types.rst

```diff
@@ -105,3 +105,93 @@ Implementers of new toolkits will likely want to write their own versions of
 these, and writers of new concrete :class:`~pyface.i_image.IImage`
 implementations may want to make use of them to simplify the implementation of
 the interface.
+
+:class:`~pyface.image.image.ImageLibrary`
+-----------------------------------------
+
+The :class:`~pyface.image_resource.ImageResource` system is built around
+supplying image files which are local to the place where they are being used.
+Sometimes, however, you want your images to be available throughout the
+application.  Pyface provides the :class:`~pyface.image.image.ImageLibrary`
```

Comment:
Umm. I'm not sure why but the `ImageLibrary` link is broken but the `ImageVolume` link below works fine in the built docs. I don't know what's going on.

Reply:
sigh. i get it now. At the end of the module, we override the name `ImageVolume` to refer to the global instance of `ImageVolume` the class.

Reply:
Yep.  It's a horrible antipattern used several places in the ETS codebase.  It would be good, but non-trivial to fix.